tools/ctl/ctl.go

```diff
@@ -462,6 +467,68 @@ var listRuns = &cobra.Command{
 	},
 }
 
+var runGCB = &cobra.Command{
+	Use:   "run-gcb -project <ID> --ecosystem <ecosystem> --package <name> --version <version> [--artifact <name>]",
+	Short: "Trigger a GCB build",
+	Args:  cobra.NoArgs,
+	Run: func(cmd *cobra.Command, args []string) {
+		r := debian.Rebuilder{}
+		mux := rebuild.RegistryMux{
+			Debian:   debianreg.HTTPRegistry{Client: http.DefaultClient},
+			CratesIO: cratesioreg.HTTPRegistry{Client: http.DefaultClient},
+			NPM:      npmreg.HTTPRegistry{Client: http.DefaultClient},
+			PyPI:     pypireg.HTTPRegistry{Client: http.DefaultClient},
+		}
+		t := rebuild.Target{
+			Ecosystem: rebuild.Ecosystem(*ecosystem),
+			Package:   *pkg,
+			Version:   *version,
+			Artifact:  *artifact,
+		}
+		s, err := r.InferStrategy(cmd.Context(), t, mux, nil, nil)
```

Comment:
what if we support local execution (with inferenceservice) as well as remote execution (with an API handler). that way you could avoid managing the Rebuilder directly. that said, we may want to add the repo storage to InferDeps (tho the default of memfs should be fine in the general case here).